src/testHelpers/src/main/kotlin/org/vaccineimpact/api/test_helpers/DatabaseCreationHelper.kt

```diff
@@ -60,37 +99,43 @@ class DatabaseCreationHelper(private val config: DatabaseConfig)
         }
     }
 
-    private fun databaseExists(dbName: String): Boolean
+    private fun databaseExists(dbName: String, maxAttempts: Int = 10): Boolean
     {
-        println("Checking that database '$dbName' exists...")
-        var attemptsRemaining = 10
+        print("Checking that database '$dbName' exists...")
+        var attemptsRemaining = maxAttempts
         while (attemptsRemaining > 0)
         {
             if (check(dbName))
             {
+                println("✔")
                 return true
             }
             else
             {
-                println("Unable to connect. I will wait and then retry $attemptsRemaining more times")
                 attemptsRemaining--
-                Thread.sleep(2000)
+                if (attemptsRemaining > 0)
+                {
+                    println("Unable to connect. I will wait and then retry $attemptsRemaining more times")
+                    Thread.sleep(2000)
+                }
             }
         }
         return false
     }
 
     fun check(dbName: String): Boolean
     {
-        try
-        {
-            config.factory(dbName).close()
-            return true
-        }
-        catch (e: UnableToConnectToDatabase)
-        {
-            error = e
-            return false
+        return temporarilyDisableLogging("org.postgresql") {
```

Comment:
What's the reason for this disable logging function?

Reply:
I've added a comment explaining.